scripts/test/SANS/gui_logic/run_tab_presenter_test.py

```diff
@@ -302,6 +302,37 @@ def test_that_populates_the_property_manager_data_service_when_processing_is_cal
 
         self._clear_property_manager_data_service()
 
+    def test_that_can_add_new_masks(self):
+        # Arrange
+        self._clear_property_manager_data_service()
+        content = "# MANTID_BATCH_FILE add more text here\n" \
+                  "sample_sans,SANS2D00022024,sample_trans,SANS2D00022048," \
+                  "sample_direct_beam,SANS2D00022048,output_as,test_file\n" \
+                  "sample_sans,SANS2D00022024,output_as,test_file2\n"
+        batch_file_path = save_to_csv(content)
+        user_file_path = create_user_file(sample_user_file)
+        view, _, _ = create_mock_view(user_file_path, batch_file_path)
+        # We just use the sample_user_file since it exists.
+        view.get_mask_file = mock.MagicMock(return_value=user_file_path)
+
```

Comment:
I have just had a quick look and the above code seems to be repeated. Is it worth creating moving it to setup or creating a setup function?